codec-http2/src/main/java/io/netty/handler/codec/http2/StreamBufferingEncoder.java

```diff
@@ -72,9 +72,9 @@ public Http2ChannelClosedException() {
         private static final long serialVersionUID = 1326785622777291198L;
         private final int lastStreamId;
         private final long errorCode;
-        private final ByteString debugData;
+        private final AsciiString debugData;
```

Comment:
We are back to the gray area of needing an immutable byte string ... is this even used? If we still need it we have a few options:

1. Leave as is ... comment it should be treated as a byte array.
2. Change to `byte[]`
3. Change to `ByteBuf` ... we would need to be careful about reference count in this case.

WDYT?